src/app/task/done-tasks/done-tasks.component.scss

```diff
@@ -0,0 +1,13 @@
+h2 {
+    margin: 10px 10px 10px 10px;
+    color: darkblue;
+}
+
+li {
+    margin: 0px 0px 0px 10px;
+}
+
+.list-group-item {
+    border-color: black;
+    width: max-content;
```

Comment:
You can use bootstrap class w-100 which makes element take 100% width of its parent

Reply:
![image](https://user-images.githubusercontent.com/45048289/75243248-d150d500-57c9-11ea-952d-634924db1567.png)
How to create a multiple class? It doesn't work in this way.. when I put it into <div>, it doesn't help

Reply:
create multiple class? What does that mean? Also, you don't have to add screenshots here. After you push you can reference specific line from file and it will be displayed here. 
https://stackoverflow.com/questions/23821235/how-to-link-to-specific-line-number-on-github

If you want to add multiple classes to html tag you would do: class="class1 class2 class3"